packages/components/src/components/dropdown/items.tsx

```diff
@@ -1,30 +1,50 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 import { getCurrencyDisplayCode } from '@deriv/shared';
 import Popover from '../popover';
 import Text from '../text';
+import { TListItem } from './utility';
+
+type TItem = {
+    className?: string;
+    handleSelect: (item: TListItem) => void;
+    has_symbol?: boolean;
+    onKeyPressed: (event: KeyboardEvent, item: TListItem) => void;
+    value?: string | number;
+    is_align_text_left?: boolean;
+    nodes?: any;
```

Comment:
@niloofar-deriv could you please try React.ReactNode[] or NodeList here instead of any?🙂

Reply:
@maryia-deriv thanks for the suggestions but none of them works disappointed